x-pack/plugins/uptime/public/components/monitor/synthetics/step_screenshot_display.tsx

```diff
@@ -30,6 +30,7 @@ const StepImage = styled(EuiImage)`
       display: none;
     }
     width: ${THUMBNAIL_WIDTH},
+    margin-right: 30px;
```

Comment:
Can we use a spacer here instead of hardcoded value? Just asking I genuinely don't know lol.